radixdlt-core/radixdlt/src/main/java/com/radixdlt/consensus/BFTHeader.java

```diff
@@ -98,30 +100,28 @@
 	@DsonOutput(Output.ALL)
 	private final LedgerHeader ledgerHeader;
 
-	BFTHeader() {
-		// Serializer only
-		this.view = null;
-		this.vertexId = null;
-		this.ledgerHeader = null;
-	}
-
 	// TODO: Move command output to a more opaque data structure
 	public BFTHeader(
 		View view, // consensus data
 		HashCode vertexId, // consensus data
 		LedgerHeader ledgerHeader
 	) {
-		this.view = view;
-		this.vertexId = vertexId;
-		this.ledgerHeader = ledgerHeader;
+		this.view = requireNonNull(view);
+		this.vertexId = requireNonNull(vertexId);
+		this.ledgerHeader = requireNonNull(ledgerHeader);
+	}
+
+	@JsonCreator
+	public static BFTHeader create(
+		@JsonProperty("view") long number,

```

Comment:
This should be `required` as well?

Reply:
This a good question. I think it was assumed to be already required as it is a primitive type, but not sure it is a correct assumption as the default value could be used instead.